extensions/wikia/Pages/PagesEntry.php

```diff
@@ -40,6 +43,8 @@ public static function newFromPageAndRevision( WikiPage $wikiPage, Revision $rev
 		$pagesEntry->latestRevisionId = $revision->getId();
 		$pagesEntry->latestRevisionTimestamp = $revision->getTimestamp();
 
+		$pagesEntry->createdAtTimestamp = Revision::getOldestRevisionTimestamp( $revision->getPage() );
```

Comment:
We could avoid a database query here: `UpdatePagesTask` will perform an `INSERT ... ON DUPLICATE KEY UPDATE` into `dataware.pages`. If it's an `INSERT` then the page was just created, and we can insert the current revision timestamp as `dataware.pages.page_created_at`. Otherwise, we do not specify `page_created_at` for `UPDATE` so changes to existing rows will not change it.

Reply:
as far as I know those data could be deleted (or don't exists atm), that's why it should be possible to retrieve created_at at any time

Reply:
I think it's better to skip reading from master here and if a maintenance script or other thing needs this info, they can make a SQL query